### PR TITLE
OSSM-4295: OSSM 2.4.1, 2.3.5, and 2.2.8 Z Stream Release Notes

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OSSM-4295

Link to docs preview: https://61885--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by luksa — APPROVED** [review]
LGTM

File: modules/ossm-rn-2-2-8.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+|1.12.9
+
+|Envoy Proxy
+|1.20.8
+
+|Jaeger
+|1.42.0
+
+|Kiali
+|1.48.6
```
Comment (pbajjuri20) [inline modules/ossm-rn-2-2-8.adoc:25]: Kiali Version is updated : 1.48.7

File: modules/ossm-rn-2-3-5.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+|1.14.6
+
+|Envoy Proxy
+|1.22.9
+
+|Jaeger
+|1.42.0
+
+|Kiali
+|1.57.9
```
Comment (pbajjuri20) [inline modules/ossm-rn-2-3-5.adoc:25]: 1.57.10

File: modules/ossm-rn-2-4-1.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+|1.16.5
+
+|Envoy Proxy
+|1.24.8
+
+|Jaeger
+|1.42.0
+
+|Kiali
+|1.65.6
```
Comment (pbajjuri20) [inline modules/ossm-rn-2-4-1.adoc:25]: 1.65.7

File: modules/ossm-rn-new-features.adoc [hunk 4]
```diff
@@ -15,6 +15,28 @@ Module included in the following assemblies:
...
+|1.16.5
+
+|Envoy Proxy
+|1.24.8
+
+|Jaeger
+|1.42.0
+
+|Kiali
+|1.65.6
```
Comment (pbajjuri20) [inline modules/ossm-rn-new-features.adoc:37]: 1.65.7

File: modules/ossm-rn-new-features.adoc [hunk 5]
```diff
@@ -146,6 +168,28 @@ spec:
...
+|1.14.6
+
+|Envoy Proxy
+|1.22.9
+
+|Jaeger
+|1.42.0
+
+|Kiali
+|1.57.9
```
Comment (pbajjuri20) [inline modules/ossm-rn-new-features.adoc:190]: 1.57.10

File: modules/ossm-rn-new-features.adoc [hunk 6]
```diff
@@ -332,6 +376,28 @@ Additionally, the SMMR must also be configured for cluster-wide deployment. This
...
+|1.12.9
+
+|Envoy Proxy
+|1.20.8
+
+|Jaeger
+|1.42.0
+
+|Kiali
+|1.48.6
```
Comment (pbajjuri20) [inline modules/ossm-rn-new-features.adoc:398]: 1.48.7

File: modules/ossm-rn-2-2-8.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-2-2-8.adoc:25]: done

File: modules/ossm-rn-2-3-5.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-2-3-5.adoc:25]: done

File: modules/ossm-rn-2-4-1.adoc (hunk 3 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-2-4-1.adoc:25]: done

File: modules/ossm-rn-new-features.adoc (hunk 4 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: done

File: modules/ossm-rn-new-features.adoc (hunk 5 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: done

File: modules/ossm-rn-new-features.adoc (hunk 6 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: done

File: modules/ossm-rn-new-features.adoc [hunk 7]
```diff
@@ -146,6 +168,28 @@ spec:
...
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.10 and later versions.
+
+=== Component versions included in {SMProductName} version 2.3.5
+
+|===
+|Component |Version
+
+|Istio
+|1.14.6
```
Comment (pbajjuri20) [inline modules/ossm-rn-new-features.adoc:181]: 1.14.5

**Review by pbajjuri20 — APPROVED** [review]

**Review by gwynnemonahan — APPROVED** [review]
lgtm

File: modules/ossm-rn-fixed-issues.adoc [hunk 8]
```diff
@@ -19,8 +19,19 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-4197[OSSM-4197] Previously if you deployed a v2.2 or v2.1 of the 'ServiceMeshControlPlane' resource, the `/etc/cni/multus/net.d/` directory was not created. As a result, the `istio-cni` pod failed to become ready, and the `istio-cni` pods log contained the following message:
```
Comment (abrennan89) [inline modules/ossm-rn-fixed-issues.adoc:22]:
```suggestion
* https://issues.redhat.com/browse/OSSM-4197[OSSM-4197] Previously, if you deployed a v2.2 or v2.1 of the 'ServiceMeshControlPlane' resource, the `/etc/cni/multus/net.d/` directory was not created. As a result, the `istio-cni` pod failed to become ready, and the `istio-cni` pods log contained the following message:
```

File: modules/ossm-rn-new-features.adoc [hunk 9]
```diff
@@ -15,6 +15,28 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4.1
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.10 and later versions.
```
Comment (abrennan89) [inline modules/ossm-rn-new-features.adoc:20]: @sheriff-rh FYI in case this overlaps with the work you're doing on version numbers?

**Review by abrennan89 — APPROVED** [review]
One minor comment @rh-tokeefe

File: modules/ossm-rn-fixed-issues.adoc (hunk 8 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-fixed-issues.adoc]: Done

File: modules/ossm-rn-new-features.adoc (hunk 9 above)
Comment (sheriff-rh) [reply modules/ossm-rn-new-features.adoc:20]: Thanks for checking @abrennan89! This module is not in my 4.14 bump PR. :+1:

**Pull request merged**